source/streaming-mode/streaming-read-config.txt

```diff
@@ -150,6 +157,8 @@ You can configure the following properties when reading a change stream from Mon
      - | Specifies whether to publish the changed document or the full
          change stream document.
        |
+       | When this setting is ``false``, you must explicitly specify a schema.
```

Comment:
Question:

Should the schema contain the fields you want to read from the change stream document? If this is the case, I think including that info could help the reader understand the action needed to take.

Reply:
will check with engineering and update if needed